src/parser/parsers/attributes.py

```diff
@@ -0,0 +1,42 @@
+import sys
+import os
+
+# bring utils module in scope
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '../..')))
+from .constants import OUTPUT_DIR
+import utils.json_utils as json_utils
+import utils.string_utils as string_utils
+
+
+class AttributeParser:
+    def __init__(self, parsed_heroes, localizations):
+        self.heroes_data = parsed_heroes
+        self.localizations = localizations
+
+    def run(self):
+        all_attributes = {}
+        all_attributes = self._parse_stats_ui(self.heroes_data)
+
+        json_utils.write(
+            OUTPUT_DIR + 'json/attribute-data.json', json_utils.sort_dict(all_attributes)
+        )
+
+    # Move StatsUI data to attributes data
```

Comment:
Minor, but I think any docs for a function should follow this convention, as it will show the comment when you hover over any use of the function. 
```python
    def _parse_stats_ui(self, heroes_data):
        """Move StatsUI data to attributes data"""
```

Reply:
Good point, will start doing this. Completed.